hw/ip/otbn/dv/otbnsim/stepped.py

```diff
@@ -260,6 +262,18 @@ def on_edn_urnd_step(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
     return None
 
 
+def on_set_keymgr_value(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
+    if len(args) != 3:
+        raise ValueError('set_keymgr_value expects exactly 1 argument. Got {}.'
+                         .format(args))
+    key0 = int(args[0], 0)
+    key1 = int(args[1], 0)
```

Comment:
Probably better to use `read_word` here, which checks argument ranges and gives better error messages if something goes wrong. I think it should be something like:
```suggestion
    key0 = read_word('key0', args[0], 384)
    key1 = read_word('key1', args[1], 384)
```
(I've just noticed that the EDN functions aren't passing quite the right names: we should probably fix that as a separate PR)

Reply:
I can't change this to read_word because it is a 384 bit variable. I get an error from read_word function saying the size is greater than 32 bits. I tried this earlier and changed it due to syntax error.

Reply:
Looks like you wrote `32` as the last argument. Try with `384` as I suggested above. There might be a problem, but it definitely shouldn't spit out a message involving the string "32".

Reply:
Yeah, I had made a mistake of writing 32 instead of 384. It works with the correction.